src/TestingComponenets/CoinsTable.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I noticed a typo in the folder name -- please change to `TestingComponents` instead of `TestingComponenets`